giraffe/src/utils/fromFlux.ts

```diff
@@ -206,31 +245,57 @@ export const fastFromFlux = (fluxCSV: string): FromFluxResult => {
     let columnKey = ''
     let columnDefault: any = ''
 
-    for (const chunk of chunks) {
-      const splittedChunk = chunk.split('\n')
+    let chunk = ''
+    for (const [start, end] of chunks) {
+      let index = 0
+      chunk = fluxCSV.substring(start, end)
 
-      const tableTexts = []
-      const annotationTexts = []
+      annotationText = ''
+      tableText = ''
 
-      splittedChunk.forEach(line => {
-        if (line.startsWith('#')) {
-          annotationTexts.push(line)
+      while (index !== -1) {
```

Comment:
note, this logic can be moved into the chunk discovery part after closing this PR